rfc/wip-tform/index.md

```diff
@@ -0,0 +1,681 @@
+regions of interest, etc. SHOULD ensure that they are in the same coordinate system (same name, with identical axes) or can be
+transformed to the same coordinate system before doing analysis. See the example below.
+
+
+### "axes" metadata
+
+"axes" describes the dimensions of a coordinate systems. It is a list of dictionaries, where each dictionary describes a dimension (axis) and:
+- MUST contain the field "name" that gives the name for this dimension. The values MUST be unique across all "name" fields.
+- SHOULD contain the field "type". It SHOULD be one of "array", "space", "time", "channel", "coordinate", or "displacement" but MAY take other values for custom axis types that are not part of this specification yet.
+- MAY contain the field "discrete". The value MUST be a boolean, and is `true` if the axis represents a discrete dimension.
+- SHOULD contain the field "unit" to specify the physical unit of this dimension. The value SHOULD be one of the following strings, which are valid units according to UDUNITS-2.
+    - Units for "space" axes: 'angstrom', 'attometer', 'centimeter', 'decimeter', 'exameter', 'femtometer', 'foot', 'gigameter', 'hectometer', 'inch', 'kilometer', 'megameter', 'meter', 'micrometer', 'mile', 'millimeter', 'nanometer', 'parsec', 'petameter', 'picometer', 'terameter', 'yard', 'yoctometer', 'yottameter', 'zeptometer', 'zettameter'
+    - Units for "time" axes: 'attosecond', 'centisecond', 'day', 'decisecond', 'exasecond', 'femtosecond', 'gigasecond', 'hectosecond', 'hour', 'kilosecond', 'megasecond', 'microsecond', 'millisecond', 'minute', 'nanosecond', 'petasecond', 'picosecond', 'second', 'terasecond', 'yoctosecond', 'yottasecond', 'zeptosecond', 'zettasecond'
+- MAY contain the field "longName". The value MUST be a string, and can provide a longer name or description of an axis and its properties.
+
+If part of multiscales metadata, the length of "axes" MUST be equal to the number of dimensions of the arrays that contain the image data.
+
+Arrays are often thought of as containing discrete samples along the continuous variable. Axes representing space and time are
+usually continuous, meaning they can be indexed by real-valued (floating point) numbers whereas discrete axes may be indexed
+only by integers. Arrays are inherently discrete (see Array coordinate systems, below) , but values "in between" discrete
+coordinates can be retreived using an *interpolation* method.  If an axis is continuous (`"discrete" : false`), it indicates
+that indexing with continuous values is meaningful, and that interpolation may be performed along that axis. Interpolation may
+be performed jointly across axes with the same `type`, and interpolation should not be performed for discrete axes or jointly
+across axes with differing `type`s.  Other non-continuous variables and axis types are also usually discrete, such as
+`channel`s, `coordinate`s, and `displacement`s.
```

Comment:
I don't think we should say that interpolation _may_ or _may not_ be performed. Generally, we should keep the normative statements restricted to the content of the metadata, rather than what actions people / software might take. We can instead make a non-normative statement to the effect that, for an axis with `{"discrete": false}` , smoothing and resampling of data values along this axis is a well-defined operation, whereas the same is not true for an axis with `{"discrete": True}`

Reply:
https://github.com/bogovicj/ngff/commit/dadff02cde9eb5dd76aa975702d9409a2d190c6e